src/commands/step-functions/helpers.ts

```diff
@@ -0,0 +1,50 @@
+import {StepFunctions} from 'aws-sdk'
+
+export const buildArn = (
+  partition: string,
+  service: string,
+  region: string,
+  accountId: string,
+  resourceType: string,
+  resourceId: string
+): string => {
+  return `arn:${partition}:${service}:${region}:${accountId}:${resourceType}:${resourceId}`
+}
+
+export const buildLogGroupName = (stepFunctionName: string, env: string | undefined): string => {
+  return `/aws/vendedlogs/states/${stepFunctionName}-Logs${env !== undefined ? '-' + env : ''}`
+}
+
+export const buildSubscriptionFilterName = (stepFunctionName: string): string => {
+  return `${stepFunctionName}LogGroupSubscription`
+}
+
+export const isValidArn = (str: string): boolean => {
+  const arnFields = str.split(':')
+
+  return arnFields.length >= 6 && arnFields[0] === 'arn'
+}
+
+export const getStepFunctionLogGroupArn = (stepFunction: StepFunctions.DescribeStateMachineOutput): string => {
+  const [logDestinations] = stepFunction.loggingConfiguration?.destinations ?? [{cloudWatchLogsLogGroup: {}}]
+
+  return logDestinations.cloudWatchLogsLogGroup?.logGroupArn ?? ''
+}
+
+export const parseArn = (
+  arn: string
+): {
+  partition: string
```

Comment:
Looks like the partition would always be `aws`? If so, we probably shouldn't need to parse this out. Same for `buildArn()`.